cmd/kubelet/kubelet.go

```diff
@@ -64,6 +64,7 @@ var (
 	oomScoreAdj             = flag.Int("oom_score_adj", -900, "The oom_score_adj value for kubelet process. Values must be within the range [-1000, 1000]")
 	apiServerList           util.StringList
 	clusterDomain           = flag.String("cluster_domain", "", "Domain for this cluster.  If set, kubelet will configure all containers to search this domain in addition to the host's search domains")
+	masterServiceNamespace  = flag.String("master_service_namespace", "default", "The namespace from which the kubernetes master services should be injected into pods")
```

Comment:
should this be master_services_namespaces since we are injecting multiple services?